core/vtk/ttkMergeTreeAutoencoder/ttkMergeTreeAutoencoder.h

```diff
@@ -0,0 +1,555 @@
+/// \ingroup vtk
+/// \class ttkMergeTreeAutoencoder
+/// \author Mathieu Pont <mathieu.pont@lip6.fr>
+/// \date 2023.
+///
+/// \brief TTK VTK-filter that wraps the ttk::MergeTreeAutoencoder module.
+///
+/// This VTK filter uses the ttk::MergeTreeAutoencoder module to compute an
+/// auto-encoder of merge trees or persistence diagrams.
+///
+/// \param Input vtkMultiBlockDataSet Input trees
+/// \param Input (optional) vtkMultiBlockDataSet Input trees
+/// \param Input (optional) vtkTable Info (such as clustering assigment)
+/// \param Output vtkMultiBlockDataSet Processed Input Trees
```

Comment:
Please move this output to the last position (it is less central than the previous 3 outputs)

Reply:
Thanks